sidebars.js

```diff
@@ -272,8 +601,608 @@ const sidebars = {
       label: 'Releases',
       collapsible: true,
       items: [
-        'releases/release-notes',
-        'releases/release-support-policy',
+        {
+          type: 'doc',
+          id: 'releases/release-notes',
+          label: 'Release Notes',
+        },
+        {
+          type: 'doc',
+          id: 'releases/release-support-policy',
+          label: 'Release Support Policy',
+        },
+      ],
+    },
+  ],
+
+  // Japanese sidebar nav
+  docsJapanese: [
+    {
+      type: 'doc',
+      label: 'ScalarDL ドキュメンテーションホーム',
+      id: 'index',
+    },
+    {
+      'type': 'category',
+      'label': 'ScalarDL について',
+      'collapsible': true,
+      'items': [
+        {
+          type: 'doc',
+          id: 'overview',
+          label: '概要',
+        },
+        {
+          type: 'doc',
+          id: 'design',
+          label: 'デザイン',
+        },
+        {
+          type: 'doc',
+          id: 'implementation',
+          label: '実装',
+        },
+        {
+          type: 'doc',
+          id: 'requirements',
+          label: '要件',
+        },
+        {
+          type: 'doc',
+          id: 'roadmap',
+          label: 'ロードマップ',
+        },
+      ]
+    },
+    {
+      type: 'category',
+      label: 'はじめよう',
+      collapsible: true,
+      items: [
+        {
+          type: 'doc',
+          id: 'getting-started',
+          label: 'ScalarDL Ledger を通じてコントラクトを実行',
+        },
+        {
+          type: 'doc',
+          id: 'getting-started-auditor',
+          label: 'ScalarDL Ledger と ScalarDL Auditor を通じてコントラクトを実行',
```

Comment:
```suggestion
          label: 'ScalarDL Auditor モードを通じてコントラクトを実行',
```
If we can use `Auditor mode` as an official word, we can make this label shorter.

Reply:
Addressed, as mentioned in the https://github.com/scalar-labs/docs-scalardl/pull/597/files#r1897305521 comment thread.